onnx/backend/test/case/model/sequence.py

```diff
@@ -0,0 +1,240 @@
+from __future__ import absolute_import
```

Comment:
Future task: we need for single operator. There are serveral tests at grahp level but we need single-node test for fine-level tests. The type of sequence can be Python list, `[x, y, z]`, where `x`, `y`, `z` are numpy tensors.